packages/design-system-tokens/src/themes/cmsgov-component.ts

```diff
@@ -449,11 +449,20 @@ export const components: AnyTokenValues = {
 
   'usa-banner': {
     '__background-color':                         t.color['gray-lightest'],
-    '__color':                                    t.color['black'],
-    '-close__background-color':                   t.color['gray-light'],
-    '-action__color':                             t.color['primary-darker'],
-    '-lock-icon__color':                          t.color['success-light'],
-    '-heading__font-size':                        '0.75rem',
+    '__color':                                    t.color['base'],
+    '__max-width':                                t.global['site-max-width'],
+    '-gutter-width':                              t.global['grid-gutter-width'],
+    '-heading__font-family':                      t.font['sans'],
+    '-heading__line-height':                      '1.1',
+    '-heading__font-size':                        t.font['size-sm'],
+    '-panel__font-family':                        t.font['sans'],
+    '-panel__line-height':                        t.font['line-height-base'],
+    '-panel__font-size':                          t.font['size-base'],
+    '-link__color':                               t.color['primary'],
+    '-domain-icon__color':                        t.color['primary'],
+    '-security-icon__color':                      t.color['success-light'],
+    '-close-button__background-color--mobile':    t.color['gray-lighter'],
+    '-close-button__color--mobile':               t.color['primary'],
   },
```

Comment:
Added more custom props than USWDS called for - mainly because I can foresee future themes wanting more customization.

USWDS calls out the following variables:
- `$theme-banner-background-color`
- `$theme-banner-font-family`
- `$theme-banner-link-color`
- `$theme-banner-max-width`

Reply:
Something to comment on here... if teams can customize their font family, it makes sense to also allow them to customize their line-heights and font-sizes. Not all fonts are built the same, so expecting a heading to have the same visual weight regardless of font-size is a little short-sighted imho.